models/migrations/v45.go

```diff
@@ -18,6 +18,7 @@ func removeIndexColumnFromRepoUnitTable(x *xorm.Engine) (err error) {
 		log.Warn("Unable to drop columns in SQLite")
 	case setting.UseMySQL, setting.UsePostgreSQL, setting.UseMSSQL, setting.UseTiDB:
 		if _, err := x.Exec("ALTER TABLE repo_unit DROP COLUMN `index`"); err != nil {
+			// Ignoring this error in case we run this migration second time (after migration reordeting)
```

Comment:
We are not ignoring this error. Also typo (reordeting)

Reply:
Then there is no way to fix this error

Reply:
At least I don't see how we can fix to run 3 migrations again that were not run for all who upgraded from 1.2 -> 1.3 without running them for all and ignoring this error

Reply:
I'm not sure what you mean; can't we just replace the return with `log.Warn`?

Reply:
@ethantkoenig I already do log.Warn this error, see code below this comment. By ignoring I meant we do not fail migration by returning error

Reply:
line 22 (directly beneath this comment) is a `return` statement

Reply:
Ooooh... Sry, just woke up and did not have my coffee yet :))) I thought you are saying this error can't be ignored but I just have added comment also to wrong file :)

Reply:
I thought you are commenting on v51 migration where there is log.Warn and same comment